### PR TITLE
[issue-13] Return non-zero status code upon error

### LINKED ISSUE #13: Installation script exits with status code zero upon missing dependencies
Refer to sdkman/sdkman-cli#709

### PR DESCRIPTION
**Pull request by ddcprg**
Fixes #13 and sdkman/sdkman-cli#709

### CODE REVIEW

File: app/views/install.scala.txt [hunk 1]
```diff
@@ -245,7 +245,7 @@ if [[ -z "$ARCHIVE_OK" ]]; then
 	echo "* easy sign up: https://slack.sdkman.io/"
 	echo "* report on channel: https://sdkman.slack.com/app_redirect?channel=user-issues"
 	rm -rf "$SDKMAN_DIR"
-	exit
+	exit 2
```
Comment (marc0der) [inline app/views/install.scala.txt]: Is there a reason why this is 2, not 1?

Comment (ddcprg) [reply app/views/install.scala.txt]: Just to differentiate between missing dependencies and error during file decompression

Comment (ddcprg) [reply app/views/install.scala.txt]: Just noticed Travis has failed, I'll take a look at the error tonight

Comment (marc0der) [reply app/views/install.scala.txt]: I think it's trying to publish an artifact to docker hub. I need to exclude the publish phase on PRs.

**Comment by marc0der** [comment]
Thanks! I'll release it to prod today.

**Comment by ddcprg** [comment]
Nice @marc0der ! Thank you!

**Comment by marc0der** [comment]
I've just released it. Can you confirm that it working as expected?

**Pull request merged**